Jenkinsfile

```diff
@@ -2,17 +2,28 @@ pipeline {
     agent {
         docker {
             label 'docker'
-            image 'gradle:4.6.0-jdk8-alpine'
+            image 'gradle:4.10.2-jdk8-alpine'
         }
     }
     stages {
         stage('Build') {
+            environment {
+                COVERALLS_REPO_TOKEN = 'upY6ITrrYihwYk9AuNHheu6XGkL8eQJP2'

```

Comment:
Is it a dangerous token?  From my understanding it's only for the Coveralls server to be able to identify the repo?

Reply:
Not completely sure, but look at this:
`Note: Do not make your repo token public.`
https://stackoverflow.com/questions/44771458/keep-token-in-yaml-file-private-secure-coveralls-io

Reply:
I've removed the token, it's apparently only required for private repositories.

Reply:
Ugh. It does not work without a token.

Creating Jenkins credentials for every repo to store the Coveralls token is no scalable solution however.

It would appear we need Travis to build repositories for the Coveralls integration to work.

Reply:
I've added back the Travis CI build and updated the repo token in the Travis project configuration.